### PR TITLE
Bumped python version to 3.5.3.

### PR DESCRIPTION
**Pull request by wheerd**
See #110.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Let us sort https://github.com/conda-forge/python-feedstock/pull/109 before merging this one.

**Comment by ocefpaf** [comment]
> Let us sort https://github.com/conda-forge/python-feedstock/pull/109 before merging this one.

Done! Can you please rebase and fix the checksum? This should be good to go after that.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
+    # Adjust FD_SETSIZE to allow select() to wait on more connections
+    # simultaneously.  Desirable for serious use of Tornado on Windows.
+    # See also:
+    # - https://bugs.python.org/issue28708
+    # - https://github.com/zeromq/libzmq/issues/1165
+    - fd_setsize.patch                # [win]
 
 build:
-  number: 4
+  number: 5
```
Comment (jjhelmus) [inline recipe/meta.yaml]: This should be set to 0. Since this is a new version we get to reset the build number.

**Comment by jjhelmus** [comment]
Thanks for the update @wheerd.  Looks great!

**Pull request merged**